.deps/dev.md

```diff
@@ -616,10 +672,14 @@
 | [`node-modules-regexp@1.0.0`](https://github.com/jamestalmage/node-modules-regexp.git) | MIT | clearlydefined |
 | [`node-notifier@7.0.1`](git+ssh://git@github.com/mikaelbr/node-notifier.git) | MIT |  |
 | [`node-object-hash@1.4.2`](git@github.com:SkeLLLa/node-object-hash.git) | ISC | clearlydefined |
+| [`node-releases@1.1.59`](git+https://github.com/chicoxyzzy/node-releases.git) | MIT | clearlydefined |
 | [`normalize-package-data@2.5.0`](git://github.com/npm/normalize-package-data.git) | BSD-2-Clause |  |
 | [`normalize-path@3.0.0`](https://github.com/jonschlinkert/normalize-path.git) | MIT | [CQ17757](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17757) |
+| [`normalize-range@0.1.2`](https://github.com/jamestalmage/normalize-range.git) | MIT | [CQ18717](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18717) |
+| [`normalize-selector@0.2.0`](git://github.com/getify/normalize-selector.git) | MIT |  |
```

Comment:
Could you provide links for all needed CQs?

Reply:
no, as it's a generated file.

Reply:
It seems I didn't get your question. 
Did you mean to add the CQ link to the `dev.md` or just asking if I created CQs for a new dependency?

Reply:
I just asking to create CQ a new dependency.

Reply:
OK, thanks, now I got it.
I'll check and create CQs if necessary and update the PR description.